<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEditFacture">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>391</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2182241398</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Invoice : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrint">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetting">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUpdateView">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>540</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>UpdateView</string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetShow" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>681</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
